client/src/components/AdvancedSearch.js

```diff
@@ -13,16 +13,23 @@ import {
   Button,
   Col,
   ControlLabel,
-  DropdownButton,
   FormControl,
   FormGroup,
-  MenuItem,
   Row
 } from "react-bootstrap"
 import { connect } from "react-redux"
 import { useHistory } from "react-router-dom"
 import REMOVE_ICON from "resources/delete.png"
 
+import {
+  Classes,
+  Menu,
+  MenuItem,
+  Popover,
+  Position as PopoverPosition,
+  PopoverInteractionKind
+} from "@blueprintjs/core"
+
```

Comment:
Please honour the correct ordering of imports, and add no blank lines between imports.